src/main/webapp/css/style.css

```diff
@@ -13,3 +17,81 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.recipe-card {
+  background-color: #F6F7F7;
+  border: groove;
+  margin: 0.5vw;
+  /* border-radius: 25px; */
+  /* TODO: figure out how to round the recipe card
+  while keeping the picture pretty */
+  width: 24vw;
+}
+
+.image-div {
+  position: relative;
+}
+.recipe-card-image {
+  width: 24vw;
+  height: 24vw;
+  overflow: hidden;
+  object-fit: cover;
+  /* border-radius: 10px; */
```

Comment:
remove this? Or is this related to the TODO above?

Reply:
related to the TODO